modules/installation-creating-aws-dns.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-dns-{context}"]
+= Creating DNS and load balancing components in AWS
+
+You must configure DNS and load balancing in Amazon Web Services (AWS) for your
+{product-title} cluster to use. The easiest way to create these components is
+to modify the provided CloudFormation template.
+
+If you need to, you can run the template multiple times within a single VPC.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. You might need to consult Red Hat support to ensure that
+your cluster initializes correctly.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+
+.Procedure
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your
+computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "HostedZoneId", <3>
+    "ParameterValue": "Z21IXYZABCZ2A4" <4>
+  },
+  {
+    "ParameterKey": "HostedZoneName", <5>
+    "ParameterValue": "example.com" <6>
+  },
+  {
+    "ParameterKey": "PublicSubnets", <7>
+    "ParameterValue": "subnet-09c2adf329727bc59" <8>
+  },
+  {
+    "ParameterKey": "PrivateSubnets", <9>
+    "ParameterValue": "subnet-0dfe1c9412a54d352" <10>
+  },
+  {
+    "ParameterKey": "VpcId", <11>
+    "ParameterValue": "vpc-055h956b1e6572be9" <12>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the
+`install-config.yaml` file.
```

Comment:
cluster name is also used when user decide to generate only ignition-configs files.